### PR TITLE
Adjusted margins to fix sidebar animation jumping

### PR DESCRIPTION
**Pull request by jamiemccue**
sidebar-animation

### CODE REVIEW

**Comment by jamiemccue** [comment]
@chasenlehara 

Fixed:

screen

File: static/sidebar.less [hunk 1]
```diff
@@ -197,8 +194,12 @@ img.social-icon-small {
 				// targets just the top level's subgroup to indent on mobile
 			}
 		}
+		.collection-group:first-of-type {
+			margin-top: 0;
+		}
 		.collection-group {
 			text-transform: uppercase;
+			margin-top: 8px;
```
Comment (chasenlehara) [inline static/sidebar.less]: This doesn’t seem right to me… shouldn’t it be 10px?

I see the difference between this PR:

Screen Shot 2019-06-17 at 4 08 34 PM

…and canjs.com:

Screen Shot 2019-06-17 at 4 08 41 PM

I also don’t see the spacing under Data Modeling fixed.

Comment (jamiemccue) [reply static/sidebar.less]: I changed to 10px which made Data Modelling line up better:

datamodelling

Comment (chasenlehara) [reply static/sidebar.less]: When I click on can-connect, the spacing between Data Modeling and Core is still wrong.

Comment (jamiemccue) [reply static/sidebar.less]: I removed the first-of-type declaration. Looks like that fixes it. 

spacing

Comment (chasenlehara) [reply static/sidebar.less]: Yup, that might fix that… but now there’s too much space when nothing is selected.

Please make the adjustments and then thoroughly test opening & closing sections, clicking on different links in the hierarchy, etc.

**Comment by jamiemccue** [comment]
Not being used.

**Pull request closed without merging**